caselaw.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>Caselaw Access Project</title>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+		<meta name="robots" content="noindex" />
+		<meta name="author" content="Your name" />
+		<meta name="description" content="Explore American Caselaw" />
+		<meta property="og:title" content="Caselaw Access Project" />
+		<meta property="og:description" content="Explore American Caselaw" />
+		<meta property="og:url" content="https://case.law/" />
+		<meta property="og:type" content="article" />
+		<meta property="og:site_name" content="Caselaw Access Projects" />
+
+		<link href="/css/global.css" rel="stylesheet" />
+		<script type="module" src="/components/cap-map.js"></script>
+		<script type="module" src="/components/cap-header.js"></script>
+		<script type="module" src="/components/cap-stats.js"></script>
+		<script type="module" src="/components/cap-section-highlight.js"></script>
```

Comment:
For my own understanding, are these module imports required for the pattern you are suggesting, or remnants from past explorations that we can safely remove?

I know we were previously considering a pattern where a router component would overtake any route, rather than only if it begins with `/caselaw`

Reply:
There isn't a way to do a route called `/caselaw` and continue to have this be an 100% portable static site. We'd have to find a way at the web server to tell it what mime type to serve for a file with no extension, which would be hosting environment-dependent.

Working on cleaning up the imports!

Reply:
How do you feel about the pattern where each page is an `index.html` file inside a dedicated directory? E.g.
```
/
/index.html
/about/index.html
/gallery/index.html
/caselaw/index.html
....
```

I've had good luck with that "just working" for "pretty URLs" with (I think?) apache, nginx, GitHub pages, and local serving with `python http.server` and `npx http-server`, though IIRC you need an edge function for S3 static site hosting.

Would that be a good compromise: optimize for pretty URLs for most setups/hosting situations, and resign ourselves to pasting "index.html" into some paths in data files if we end up with a serving/hosting situation that doesn't support it at some point?

I'm new to the project, obviously: definitely not trying to intrude on already-made design decisions 😃 .

Reply:
Edit: I don't know why I thought you needed an edge function for S3... if I'm reading [their docs](https://docs.aws.amazon.com/AmazonS3/latest/userguide/IndexDocumentSupport.html#IndexDocumentsandFolders) correctly, this pattern works just fine there too:

> If you create a folder structure in your bucket, you must have an index document at each level. In each folder, the index document must have the same name, for example, index.html. When a user specifies a URL that resembles a folder lookup, the presence or absence of a trailing slash determines the behavior of the website. For example, the following URL, with a trailing slash, returns the photos/index.html index document.